### DISCUSSION TITLE
Problema di inizializzazione Home Assistant su Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by angelo3004** [indomus/forum#2949] [Hardware & Firmware]
Salve a tutti, sono nuovo e volevo esporvi un problema riguardo le prime fasi di homeassistant.
Un anno fa con un raspberry pi4 installai homeassistant su micro sd tramite metodo classico (con balena etcher) ma causa esami lasciai il progetto

Ieri ho voluto ricominciare ma mi sono dimenticato username e password cosi ho reinstallato homeassistant sul raspberry sempre con questo metodo ma non mi compare piu la pagina iniziale (per intenderci quella di configurazione dove ti chiede di creare username e password iniziali per creare il profilo)

Ho provato a reinstallare il sistema ma il problema continua, sapete aiutarmi?

p.s. il raspberry è collegato LAN, pensavo fosse un problema di rete ma scannerizzandola lo trovo

Grazie in anticipo

**Comment by KC-inDomus** [comment]
tramite:
- http://homeassistant.local:8123
- http://indirizzo_ip_del_raspberry:8123

DEVE rispondere. se non lo fa, non è in esecuzione.
